py/torch_tensorrt/dynamo/compile.py

```diff
@@ -42,6 +41,8 @@
     to_torch_tensorrt_device,
 )
 
+import torch_tensorrt
```

Comment:
is this required ?

Reply:
The import itself is used on the following lines:
https://github.com/pytorch/TensorRT/blob/8df34b8a1a1ad7b2b2574208b2126c6d0704e38d/py/torch_tensorrt/dynamo/compile.py#L100-L108
Regarding the import line change, it was auto-completed by the linter.